css-positioning.tar/css-positioning/absolute.css

```diff
@@ -0,0 +1,43 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top:30px; right:10px;
+
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  bottom:10px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(43, 70, 109, 0.42));
+
+  /* your code here */
+  background: linear-gradient(transparent, hsla(216, 24%, 40%, 0.42));
```

Comment:
not quite! You'll need a combination of `position: aboslute` and `top/right/bottom/left` with the right values to get the gradient to appear. You'll notice on line 39 you already have the gradient setup for you.